### PR TITLE
[SI-552] REVERT front end changes, keeps cypress tests

### PR DESCRIPTION
**Pull request by a6software**
Roll forward revert.

Keeps the Cypress tests - 1 change made, to adapt the displayed due date to match the reverted behaviour. 

Previous merged PR for reference: https://github.com/ministryofjustice/offender-categorisation/pull/804/files#diff-5f0bde0bedc16dd071ece5fe99558910d6aff24ed4490d16a4e255cef8d0e249R73

### CODE REVIEW

File: server/data/formClient.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ const selectClause = `select id,
                     security_reviewed_date as "securityReviewedDate",
                     security_reviewed_by   as "securityReviewedBy",
                     approval_date          as "approvalDate",
-                    due_by_date            as "dueByDate",
```
Comment (a6software) [inline server/data/formClient.js:18]: reverted line: https://github.com/ministryofjustice/offender-categorisation/pull/804/files#diff-280e2cf629acf221eae48557f53eda361f82f1982a63389b38488b7bd765fea2R18

File: server/services/offendersService.js [hunk 2]
```diff
@@ -756,16 +755,14 @@ module.exports = function createOffendersService(
...
         return {
           ...nomisRecord,
           displayName: `${properCaseName(nomisRecord.lastName)}, ${properCaseName(nomisRecord.firstName)}`,
           displayStatus: calculateRecatDisplayStatus(decorated.displayStatus),
           dbStatus: decorated.dbStatus,
           reason,
-          nextReviewDateDisplay: dateConverter(nextReviewDate),
-          overdue: isOverdue(nextReviewDate),
+          nextReviewDateDisplay: dateConverter(nomisRecord.nextReviewDate),
+          overdue: isOverdue(nomisRecord.nextReviewDate),
```
Comment (a6software) [inline server/services/offendersService.js:765]: reverted file reference: https://github.com/ministryofjustice/offender-categorisation/pull/804/files#diff-b051c60930dc9afcedadb930a76be4e25281f3a05bbdd8d86c9da00e565dbdecR719

File: server/views/pages/recategoriserHome.html [hunk 3]
```diff
@@ -61,8 +61,7 @@ <h1 class="govuk-heading-l">Category reviews for prisoners</h1>
         {% if row.overdue %}
         {{ govukTag({
             text: 'OVERDUE',
-            classes: 'moj-tag--red',
-            attributes: { 'title': 'Due date: ' + row.nextReviewDateDisplay }
+            classes: 'moj-tag--red'
```
Comment (a6software) [inline server/views/pages/recategoriserHome.html:64]: reverted line: https://github.com/ministryofjustice/offender-categorisation/pull/804/files#diff-dcffc61bd95e75c8dc2715247a54f5d79126d340f9b81078a2f9a217dc388bb0R65

File: test/data/formClient.test.js [hunk 4]
```diff
@@ -34,7 +34,6 @@ describe('getFormDataForUser', () => {
                     security_reviewed_date as "securityReviewedDate",
                     security_reviewed_by   as "securityReviewedBy",
                     approval_date          as "approvalDate",
-                    due_by_date            as "dueByDate",
```
Comment (a6software) [inline test/data/formClient.test.js:37]: reverted line: https://github.com/ministryofjustice/offender-categorisation/pull/804/files#diff-a4ab4e042198ae6f715ec5a798f5086ff4a19cbc671b912b63cdaa4401e5a36dR37

File: server/views/pages/supervisorHome.html [hunk 5]
```diff
@@ -70,7 +70,7 @@ <h1 class="govuk-heading-l">Prisoner Categorisation Approvals</h1>
       <td class="govuk-table__cell"><a target="_blank" href="{{ row.offenderNo | offenderLink }}">{{ row.displayName }}</a></td>
       <td class="govuk-table__cell">{{ row.offenderNo }}</td>
       <td class="govuk-table__cell">{{ row.daysSinceSentence }}</td>
-      <td class="govuk-table__cell">{{ row.dateRequired or dateConverter(row.dbRecord.dueByDate) }}</td>
+      <td class="govuk-table__cell">{{ row.dateRequired }}</td>
```
Comment (a6software) [inline server/views/pages/supervisorHome.html:73]: reverted line: https://github.com/ministryofjustice/offender-categorisation/pull/804/files#diff-5f0bde0bedc16dd071ece5fe99558910d6aff24ed4490d16a4e255cef8d0e249R73

**Review by Stubbs — APPROVED** [review]
LGTM

**Pull request merged**